src/PresentationalComponents/LandingPage/PermissionDenied.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { InfoCircleIcon } from '@patternfly/react-icons';
+import { EmptyState, EmptyStateVariant, EmptyStateIcon, EmptyStateBody, Title } from '@patternfly/react-core';
+
+const PermissionDenied = () => {
+    return (
+        <EmptyState variant={ EmptyStateVariant.large } data-testid="empty-state-denied">
+            <EmptyStateIcon icon={ InfoCircleIcon } />
+            <Title headingLevel="h4" size="lg">
+            Image Builder is not quite ready
+            </Title>
+            <EmptyStateBody>
+            Image Builder is still very early in development and not quite ready for general use yet.
+            Would you be interested in testing it once it reaches beta, let us know and we&apos;ll be in touch once it&apos;s ready!
```

Comment:
I think I'd say "wider use" instead of "general" and drop the "very": "Image Builder is still early in development and not quite ready for wider use yet."

And maybe "if you're interested" instead of "would you be"? Not sure though.

My first idea was to ask people to use the pendo feedback button in the bottom right corner, but it doesn't seem to work. @andreasn do you have an alternative idea?

Reply:
I think a blue button in the middle of the page would work best, labelled `Sign up`
The very easiest and straight forward would be to link it to a Google form, but I'll ask in the Pendo slack channel if it's possible to use the feedback form.

Reply:
Would you make it start out like a question, but ends up being a statement, so I would suggest something like this:
`If you're interested in trying it out once it reaches beta, fill out your contact information in the sign up form, and we'll be in touch once it's ready.`

Reply:
Or if we use the same language as the form Margot is going to set up:
`If you're interested in trying it out once it reaches beta, fill out your contact details and we’ll reach out to you once the service is ready!`